lib/elixir/lib/enum.ex

```diff
@@ -2278,7 +2278,8 @@ defmodule Enum do
   Sorts the `enumerable` by the given function.
 
   This function uses the merge sort algorithm. The given function should compare
-  two arguments, and return `true` if the first argument precedes the second one.
+  two arguments, and return `true` if the first argument is **less than or equal to**
+  the second one.
```

Comment:
I think the previous wording is better you are not necessarily sorting based less / greater than. How about this instead?

The given function should compare two arguments, and return `true` if the first argument precedes or is in the same place as the second one.